src/highlevel_api.jl

```diff
@@ -306,6 +392,6 @@ The quality of the result might depend on the used terminal and might vary betwe
 **Note:** The specific visualization may change between minor versions.
 """
 function topology_graphical()
-    run(`$(lstopo_no_graphics()) --no-io --no-legend --of txt`)
+    run(`$(lstopo_no_graphics()) --no-legend --of txt`)
```

Comment:
Maybe add an `io` keyword argument based on which we add `--no-io` or not? Sometimes one doesn't care about IO components and they can very much clutter the visualization.

Reply:
Done. I'm using `get_io` which is the same kwarg as with `gettopology` and `topology_init`